### PR TITLE
More pylint fixes

### PR DESCRIPTION
**Pull request by marksmayo**
Including some newly introduced ones when I rebased

### CODE REVIEW

**Comment by Miksus** [comment]
Thanks! I'll try to check and merge before the end of the week. Sorry for the delay.

File: docs/code/conds/api/cron_kwargs.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 @app.task(cron(minute="*/5"))
 def do_simple():
     "Run at every 5th minute"
-    ...
+
```
Comment (Miksus) [inline docs/code/conds/api/cron_kwargs.py:6]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/snippets/custom_condition.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def file_exists(file):
 @app.task(daily & file_exists("data.csv"))
 def do_things():
     "Task that runs once a day when data.csv exists"
-    ...
```
Comment (Miksus) [inline docs/code/snippets/custom_condition.py:15]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demos/intermediate.py [hunk 3]
```diff
@@ -7,33 +7,31 @@
...
 def do_after_another(arg=Return(do_daily)):
     """This task runs after 'do_daily' and it has its the
     return argument as an input"""
-    ...
+
 
 @app.task(daily)
 def do_with_params(arg1=FuncArg(lambda: ...), arg2=Arg("myparam")):
     """This task runs with variety of arguments"""
-    ...
```
Comment (Miksus) [inline docs/code/demos/intermediate.py:36]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demos/intermediate.py [hunk 4]
```diff
@@ -7,33 +7,31 @@
...
+
     return ...
 
 
 @app.task(after_success(do_daily))
 def do_after_another(arg=Return(do_daily)):
     """This task runs after 'do_daily' and it has its the
     return argument as an input"""
-    ...
```
Comment (Miksus) [inline docs/code/demos/intermediate.py:31]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demos/intermediate.py [hunk 5]
```diff
@@ -7,33 +7,31 @@
...
 def do_daily():
     "This task runs once a day when foo is true"
-    ...
     return ...
 
 @app.task((daily.at("10:00") | daily.at("19:00")) & time_of_week.between("Mon", "Fri"),
           execution="process")
 def do_complex():
     "This task runs on complex interval and on separate process"
-    ...
```
Comment (Miksus) [inline docs/code/demos/intermediate.py:23]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demos/intermediate.py [hunk 6]
```diff
@@ -7,33 +7,31 @@
 @app.cond()
 def is_foo():
     "This is a custom condition"
-    ...
     return True
 
 @app.task(daily & is_foo)
 def do_daily():
     "This task runs once a day when foo is true"
-    ...
```
Comment (Miksus) [inline docs/code/demos/intermediate.py:16]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demos/intermediate.py [hunk 7]
```diff
@@ -7,33 +7,31 @@
 @app.cond()
 def is_foo():
     "This is a custom condition"
-    ...
```
Comment (Miksus) [inline docs/code/demos/intermediate.py:10]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demos/advanced.py [hunk 8]
```diff
@@ -40,18 +40,15 @@ def do_pipeline(arg1=Return('do_on_process'),
...
 @app.task('daily & is foo', execution="thread")
 def do_custom():
     """This task runs once a day and when is_foo returns True
     This task runs on separate thread"""
-    ...
 
 @app.task('(true & true) | (false & True & ~True)')
 def do_complex():
     """Notice the logical expression in the task start condition"""
-    ...
```
Comment (Miksus) [inline docs/code/demos/advanced.py:54]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demos/advanced.py [hunk 9]
```diff
@@ -40,18 +40,15 @@ def do_pipeline(arg1=Return('do_on_process'),
     Argument 'arg1' gets the return value of 'do_on_process'
     Argument 'arg2' gets the return value of function 'get_item'
     Argument 'arg3' is simply the value of a session parameter 'my_arg'"""
-    ...
 
 @app.task('daily & is foo', execution="thread")
 def do_custom():
     """This task runs once a day and when is_foo returns True
     This task runs on separate thread"""
-    ...
```
Comment (Miksus) [inline docs/code/demos/advanced.py:49]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demos/advanced.py [hunk 10]
```diff
@@ -40,18 +40,15 @@ def do_pipeline(arg1=Return('do_on_process'),
     Argument 'arg1' gets the return value of 'do_on_process'
     Argument 'arg2' gets the return value of function 'get_item'
     Argument 'arg3' is simply the value of a session parameter 'my_arg'"""
-    ...
```
Comment (Miksus) [inline docs/code/demos/advanced.py:43]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demo.py [hunk 11]
```diff
@@ -6,19 +6,19 @@
...
     "This function runs twice a day (in the morning and in the afternoon)"
     # The '|' means OR operator. Fully supports logical operations.
-    ...
+
 
 @app.task("after task 'do_daily'")
 def do_after_another(arg=Return('do_daily')):
     "Run after 'do_daily' task"
     # The parameter 'arg' has the return value of the function 'do_daily'
-    ...
```
Comment (Miksus) [inline docs/code/demo.py:21]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demo.py [hunk 12]
```diff
@@ -6,19 +6,19 @@
...
 def do_daily():
     "This function runs once a day"
-    ...
+
 
 @app.task('daily between 07:00 and 10:00 | daily between 16:00 and 20:00')
 def do_twice_a_day():
     "This function runs twice a day (in the morning and in the afternoon)"
     # The '|' means OR operator. Fully supports logical operations.
-    ...
```
Comment (Miksus) [inline docs/code/demo.py:15]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/demo.py [hunk 13]
```diff
@@ -6,19 +6,19 @@
 @app.task('daily')
 def do_daily():
     "This function runs once a day"
-    ...
```
Comment (Miksus) [inline docs/code/demo.py:9]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/conds/api/crontime_kwargs.py [hunk 14]
```diff
@@ -13,4 +13,3 @@ def do_complex():
         - On 1st, 2nd and 3rd day of month
         - From February to August every second month
     """
-    ...
```
Comment (Miksus) [inline docs/code/conds/api/crontime_kwargs.py:16]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/conds/api/crontime_kwargs.py [hunk 15]
```diff
@@ -3,7 +3,7 @@
 @app.task(crontime(minute="*/5"))
 def do_simple():
     "Run at every 5th minute"
-    ...
```
Comment (Miksus) [inline docs/code/conds/api/crontime_kwargs.py:6]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/conds/api/cron_kwargs.py [hunk 16]
```diff
@@ -13,4 +13,3 @@ def do_complex():
         - On 1st, 2nd and 3rd day of month
         - From February to August every second month
     """
-    ...
```
Comment (Miksus) [inline docs/code/conds/api/cron_kwargs.py:16]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

File: docs/code/conds/api/cron.py [hunk 17]
```diff
@@ -7,4 +7,3 @@ def do_minutely():
 @app.task(cron('*/2 12-18 * Oct Fri'))
 def do_complex():
     "Run at every 2nd minute past every hour from 12 through 18 on Friday in October."
-    ...
```
Comment (Miksus) [inline docs/code/conds/api/cron.py:10]: I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).

**Review by Miksus — CHANGES_REQUESTED** [review]
Just some (perhaps opinionated) suggestions to leave the "..." in some code snippets in the documentation to indicate a placeholder and there should be more code in practice.

For example:
```python
def do_things():
    "This does things"
    ...
```
Vs:
```python
def do_things():
    "This does things"
```

**Comment by Miksus** [comment]
Sorry it took almost a month to review (had a bad week, a busy week and a small side project). I think all of those changes are appropriate but I think some of those placehoder ``...`` could be left in the snippets in the docs.

**Review by marksmayo — COMMENTED** [review]
Think I've made them all

**Comment by marksmayo** [comment]
Still says changes requested but I think I've resolved all? Just pulling in latest from your repo to see if I've missed any new ones

**Review by Miksus — APPROVED** [review]

**Comment by Miksus** [comment]
Yep, looks good. Thanks again for the fixes and sorry for the immense delay. I now booked an hour every week to go through PRs and issues so hopefully, there won't be the same delay anymore.

**Comment by marksmayo** [comment]
All good. I'm still learning so in no hurry.

**Pull request merged**